test/test_suite.py

```diff
@@ -837,3 +835,91 @@ def test_nolength_binary(self):
 @pytest.mark.skip("Spanner doesn't support quotes in table names.")
 class QuotedNameArgumentTest(_QuotedNameArgumentTest):
     pass
+
+
+@pytest.mark.skip("Spanner doesn't support IS DISTINCT FROM clause")
+class IsOrIsNotDistinctFromTest(_IsOrIsNotDistinctFromTest):
+    pass
+
+
+class OrderByLabelTest(_OrderByLabelTest):
+    @pytest.mark.skip(
+        "Spanner throws a syntax error as group by composed doesn't support."
```

Comment:
I'm not quite sure what isn't supported but perhaps it should be reworded to:
```Spanner doesn't support <unsupported statement>.```

Similar to the above skip